### ISSUE TITLE
[Idea] Remove powder from Trendy Game?

### ISSUE THREAD

**Issue by muffinjets** [daid/LADXR#93]
Either that, or maybe shuffle the Trendy Game items, but last I heard, that was really difficult.

I think having powder not be guaranteed (And maybe only have one or two powder items in the pool) will add a little bit more depth to the early game.

**Comment by foxsouns** [comment]
Never have I ever successfully obtained "Trendy Powder"
but it isn't ever in logic to obtain trendy powder, it's just an option

**Comment by muffinjets** [comment]
Of course, when I say "guaranteed" I don't mean logically guaranteed, I'm aware it's not logically expected

That being said, it's easy to learn how to do the Trendy Game and even easier to get the powder once you know

**Comment by foxsouns** [comment]
Ah, I see. Randomize the conveyor items as a whole, got it.
